guides/walkthroughs/how-to-complete-stellar-toml.md

```diff
@@ -28,7 +25,7 @@ For example, the Stellar app, [StellarX](http://stellarx.com/), uses the stellar
 
 *Many won't list your token at all without a robust stellar.toml.*
 
-The stellar.toml file is *so* important that the first Stellar Ecosystem Proposal is devoted to outlining what it should contain. You can find the complete SEP 0001 [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md), but we'll summarize the important parts below.
+The stellar.toml file is *so* important that the first Stellar Ecosystem Proposal is devoted to outlining what it should contain. You can find the complete SEP 0001 [here](../../../../../stellar-protocol/blob/master/ecosystem/sep-0001.md), but we'll summarize the important parts below.
```

Comment:
The previous links should stay. This is compiled to https://www.stellar.org/developers/ so everything outside this site should be an absolute link.

Reply:
Thanks for the clarification. 👍